fpga_interchange/dedicated_interconnect.h

```diff
@@ -31,6 +31,14 @@
 
 NEXTPNR_NAMESPACE_BEGIN
 
+enum SinkOnNetStatus
```

Comment:
I'd recommend adding a prefix to these enums, to avoid conflicts with anything else (`VALID` in particular being a common name).